src/style.css

```diff
@@ -0,0 +1,20 @@
+
+body {
+  margin: 0;
+}
+
+div {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
"earthhuman41 4 hours ago
i think it's bad idea to make this class, better to create common class for all divs and get them height 100px" 
Не понимаю, создать новый класс или прописать стиль для div?

Reply:
create new class, dont style tags